code/game/objects/items/bodybag.dm

```diff
@@ -108,41 +118,49 @@
 		overlays.Cut()
 
 
-/obj/structure/closet/bodybag/closet_special_handling(mob/living/mob_to_stuff) // overriding this
+/obj/structure/closet/bodybag/closet_special_handling(mob/living/carbon/human/mob_to_stuff) // overriding this
```

Comment:
This will get called on any living and will likely runtime if you're accessing human stuff

Reply:
The first check is a type check.

Reply:
That still breaks the practice that arguments in procs should be the same, especially since it's `ishuman` rather than `istype`
```suggestion
/obj/structure/closet/bodybag/closet_special_handling(mob/living/mob_to_stuff) // overriding this
```